priv/static/phoenix_live_reload.js

```diff
@@ -41,6 +43,11 @@ var cssStrategy = function(){
 
 var pageStrategy = function(chan){
   chan.off('assets_change');
+
+  if (restoreScrollOnReload) {
+    sessionStorage.setItem(SESSION_STORAGE_SCROLL_Y_KEY, window[targetWindow].scrollY);
+  }
```

Comment:
We set the item in a JIT-fashion, and it gets pulled from sessionStorage and cleared on the subsequent page load when the channel is joined.

Reply:
`sessionStorage` is local to a tab/window, unlike `localStorage`, so this value will be unique per tab if the user has many tabs open during development.